sample/src/androidTest/java/com/schibsted/spain/barista/sample/PermissionGranterTest.kt

```diff
@@ -49,21 +49,21 @@ class PermissionGranterTest {
     @Test
     fun ignores_already_granted_permission() {
         launchActivity {
-            verifyPermissionNotGranted(SIMPLE_PERMISSION_2)
-            requestPermission(SIMPLE_PERMISSION_2)
-            PermissionGranter.allowPermissionsIfNeeded(SIMPLE_PERMISSION_2)
-            verifyPermissionGranted(SIMPLE_PERMISSION_2)
+            verifyPermissionNotGranted(CAMERA_PERMISSION)
+            requestPermission(CAMERA_PERMISSION)
+            PermissionGranter.allowPermissionOneTime(CAMERA_PERMISSION)
+            verifyPermissionGranted(CAMERA_PERMISSION)
 
-            PermissionGranter.allowPermissionsIfNeeded(SIMPLE_PERMISSION_2)
+            PermissionGranter.allowPermissionsIfNeeded(CAMERA_PERMISSION)
         }
     }
 
 
 }
 
 // We can't reuse permission from one test to another, because they stay granted after each test
-private const val SIMPLE_PERMISSION_1 = Manifest.permission.READ_CONTACTS
```

Comment:
Oh! I explicitly named the permissions like this for a reason 😬 
As the comment says, each test needs to use a different permission, that's why I numbered them. I thought using the real name would make it easier to accidentally reuse them.

Reply:
Renamed